src/components/CopyButton.js

```diff
@@ -0,0 +1,83 @@
+/* global chrome */
+import React, { useState, useEffect } from 'react';
+import IconButton from './IconButton';
+import SuccessIcon from './icons/SuccessIcon';
+import CopyIcon from './icons/CopyIcon';
+
+const IS_DEVTOOL = !!(window.chrome && chrome.runtime && chrome.runtime.id);
```

Comment:
```suggestion
const IS_DEVTOOL = Boolean(window?.chrome?.runtime?.id);
```

Reply:
Nope, this will break. Optional chaining doesn't cover `null` cases. As null is a valid value.

Also, I want to ask you (@MichaelDeBoey) to stop with the syntax only review comments. Without a supporting comment, it feels a bit hostile. 

Besides, we have a linter. And the linter didn't flag this. Thereby, I can only see this as pushing a personal preference.

@marcosvega91, please leave this condition the way it is.

Reply:
I have reverted the change